cluster/addons/metadata-proxy/README.md

```diff
@@ -0,0 +1,5 @@
+# Metadata proxy
+==============
+
+This metadata proxy returns a 403 for kubelet's secrets, but otherwise allows
```

Comment:
"secrets" is probably misleading here. Maybe say that it returns a 403 for the configuration data stored in kube-env.